commons/test/TypeSerializer.test.ts

```diff
@@ -2,9 +2,9 @@ import Puppet from '@secret-agent/puppet';
 import Chrome83 from '@secret-agent/emulate-chrome-83';
 import TypeSerializer, { stringifiedTypeSerializerClass } from '../TypeSerializer';
 import { CanceledPromiseError } from '../interfaces/IPendingWaitEvent';
-import Log from '../Logger';
+import logger from '../Logger';
```

Comment:
Doesn't this go against our convention of naming the import variable the same name as the file?

Reply:
Yeah.. I exported the class "Log" from this class, and suddenly eslint started complaining about 3 files where we did things like this. All our logger imports go against several conventions. We use this code everywhere, but the default export is actually a logger function, which should be named createLogger. I think at some point we just need a quick cleanup of the logger exports and imports.

`import Log from '@secret-agent/commons/Logger';`